scripts/pyproject.toml

```diff
@@ -5,7 +5,10 @@ ignore = [
   "INP001",  # implicit-namespace-package
   "PLR2004", # magic-value-comparison
```

Comment:
Let's turn off all of the `pylint` rules please.

Reply:
I really don't think turning off all `pylint` rules is a good idea ... we should eat our own dogfood and that includes not blindly disabling a whole linter.